qi-lib/list.rkt

```diff
@@ -2,7 +2,7 @@
 
 ;; Upon instantiation of the module it define-and-register-pass for
 ;; deforestation
-(require racket/list
-         "flow/core/deforest.rkt")
+(require "flow/core/compiler/0100-deforest.rkt"
+         "flow/core/compiler/deforest/bindings.rkt")
 
-(provide (all-from-out racket/list))
+(provide (all-from-out "flow/core/compiler/deforest/bindings.rkt"))
```

Comment:
Nice. FTR, I do not see these numerous list-oriented bindings as belonging in `qi-lib` in the long term but rather in `qi-list`. Having them behind `(require qi/list)` seems a sufficient abstraction for this for now, but it would still set my heart at ease to move these out sooner rather than later 😉